src/components/OpenLayersMap.js

```diff
@@ -168,7 +179,7 @@ class OpenLayersMap extends React.Component{
 
     // Orientation
     var projection = Projection.get('EPSG:3857');
-    var killingtonCoords = [-72.803584,43.619210];
+    var killingtonCoords = [-106.553668, 39.612616];
```

Comment:
Put this back to killington coordinates? Or we can just make Beaver Creek the default location too and change this name, doesn't matter since there aren't any default trails or hydrants anymore.

Reply:
I figured it doesn't really matter since the default shouldn't be any mountain, but should be the central US or something. Changed it for a change of scenery and..see last sentence of PR message.

Reply:
We should change that variable to centerCoords or something though. I'll do that in my next commit.